installer/Controller/EnvController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+
+namespace Installer\Controller;
+
+
+class EnvController implements Controller
+{
+
+	/**
+	 * @return array
+	 */
+	public function do()
+	{
+
+		if (isset($_POST['envConfig']))
+		{
+			file_put_contents('.env',$_POST['envConfig']);
```

Comment:
This results in ugly DOS-style CRLF line terminators, apparently per HTML specs. I guess you should filter out `\r` characters before writing?